openedx/core/djangoapps/content_libraries/rest_api/containers.py

```diff
@@ -303,6 +302,8 @@ class LibraryContainerCollectionsView(GenericAPIView):
     """
     View to set collections for a container.
     """
+    serializer_class = serializers.DummySerializer
```

Comment:
We are using this serializer for the input data:
```
serializer = serializers.ContentLibraryItemCollectionsUpdateSerializer(data=request.data)
```

Can we use this for the serializer_class?

Reply:
Also can we have define custom swagger response?